### PR TITLE
fix #36199 m.7ktw.com

### LINKED ISSUE #36199: m.7ktw.com
### Issue URL (Incorrect Blocking)
[https://m.7ktw.com/mulu/41398/46.html](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fm.7ktw.com%2Fmulu%2F41398%2F46.html)
### Comment
Username: @BlackZeroPM
> Can’t block the ads

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>
<details>
  <summary>Screenshot 2</summary>

  Screenshot 2
</details>

### System configuration

Information | value
--- | ---
Platform: | iOS
AdGuard version: | 2.1.1
Browser: | Safari
System wide filtering: | disabled
Simplified filters: | disabled
AdGuard DNS: | Other - OpenDNS Home
Filters: | AdGuard Mobile Ads,<br>AdGuard Safari,<br>AdGuard Annoyances,<br>AdGuard Simplified domain names,<br>AdGuard Base,<br>EasyList,<br>Adblock Warning Removal List,<br>ABPVN List,<br>Fanboy Anti-Facebook List,<br>AdGuard Tracking Protection,<br>AdGuard Social Media

### PR DESCRIPTION
**Pull request by zloyden**
https://github.com/AdguardTeam/AdguardFilters/issues/36199

### CODE REVIEW

File: EnglishFilter/sections/foreign.txt [hunk 1]
```diff
@@ -785,6 +785,8 @@ kliuki.bg###container > div[style="float:left;"] > div[style]
 ! https://github.com/AdguardTeam/AdguardFilters/issues/35180
 sohu.com##div[data-spm-content="promotion"]
 sohu.com##a[href^="https://m.sohu.com/promotion?"]
+! https://github.com/AdguardTeam/AdguardFilters/issues/36199
+||*.gif$third-party,domain=m.7ktw.com
```
Comment (BlazDT) [inline EnglishFilter/sections/foreign.txt]: Do the domains for the ad change? I did not see a valid .gif when investigating the site.

So far sp0.baidu.com seems to include a tracking gif which is blocked with given rule.

Comment (zloyden) [reply EnglishFilter/sections/foreign.txt]: > Do the domains for the ad change?

I didn't see that. Found 2 domains `https://img.prscnz.cn`and `https://img.xiwnu.cn`. My rule is correct or i should change it?

Comment (BlazDT) [reply EnglishFilter/sections/foreign.txt]: Leave it that way I think - both domains show me forbidden when visiting them and domain itself is not responding.

**Review by BlazDT — APPROVED** [review]

File: EnglishFilter/sections/foreign.txt (hunk 1 above)
Comment (Alex-302) [reply EnglishFilter/sections/foreign.txt]: `||*` is unnecessary in this case

**Review by Alex-302 — CHANGES_REQUESTED** [review]

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**